scripts/register-cluster-with-dynatrace.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env sh
+
+VAULT_NAME=$1
+DYNATRACE_INSTANCE=$2
+
+DYNATRACE_CLUSTERROLE_BINDING=roles/dynatrace-cluster-role-binding.yaml
+
+K8S_API_URL=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
+CLUSTER_NAME=$(kubectl config view --minify -o jsonpath='{.clusters[0].name}')
+
+BEARER_TOKEN=$(kubectl get secret $(kubectl get sa dynatrace-monitoring \
+  -o jsonpath='{.secrets[0].name}' -n monitoring) -o jsonpath='{.data.token}' \
+  -n monitoring | base64 --decode)
+
+API_TOKEN=$(az keyvault secret show \
+  --name dynatrace-cluster-registration-token-london-${DYNATRACE_INSTANCE} \
```

Comment:
this name is very specific, could we at least drop london?
```suggestion
  --name dynatrace-cluster-registration-token-${DYNATRACE_INSTANCE} \
```